ft/onto/base_ontology.py

```diff
@@ -308,7 +312,12 @@ class PredicateLink(Link):
     ParentType = PredicateMention
     ChildType = PredicateArgument
 
-    def __init__(self, pack: DataPack, parent: Optional[Entry] = None, child: Optional[Entry] = None):
+    def __init__(
```

Comment:
why there are so many changes to the `base_ontology` file, it should be generated and all the rest should be untouched.